lib/brexit_checker/actions.yaml

```diff
@@ -1323,3 +1323,41 @@ actions:
   criteria:
   - forestry
   audience: business
+- action_id: S030
+  priority: 5
+  title: Exchange your UK licence for a licence issued by the EU country you live
+    in
+  title_url: https://europa.eu/youreurope/citizens/vehicles/driving-licence/driving-licence-renewal-exchange/index_en.htm#shortcut-0-exchanging-your-licence
+  consequence: You will not be able to drive in the EU with a UK licence after Brexit.
+  lead_time: Do it as soon as possible
+  guidance_prompt: Read the guidance
+  guidance_link_text: 'Driving in the EU after Brexit: driving licence exchange'
+  guidance_url: https://www.gov.uk/guidance/driving-in-the-eu-after-brexit-driving-licence-exchange
+  criteria:
+  - any_of:
+    - all_of:
+      - nationality-uk
+      - living-eu
+    - all_of:
+      - nationality-uk
+      - living-ie
```

Comment:
Nit: this could be simplified to 
```yaml
- all_of:
  - nationality-uk
  - any_of:
    - living-eu
    - living-ie
```

Reply:
Same with action below

Reply:
Thanks - I've updated in the spreadsheet.